libs/hpc-ui/src/lib/components/form-fields/date-picker.tsx

```diff
@@ -0,0 +1,97 @@
+import { useField } from 'formik';
+import { TextField, Link } from '@mui/material';
+import { DatePicker as BaseDatePicker } from '@mui/x-date-pickers/DatePicker';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import tw from 'twin.macro';
+import dayjs from '../../i18n/utils/dayjs';
+import { useEffect, useState } from 'react';
+
+const StyledDatePicker = tw.div`
+flex
+w-full
+items-baseline
+`;
+
+const StyledLink = tw(Link)`
+ms-8
+`;
```

Comment:
Would definitely be nicer if you applied consistent formatting in these
```suggestion
const StyledDatePicker = tw.div`
  flex
  w-full
  items-baseline
`;

const StyledLink = tw(Link)`
  ms-8
`;
```